### ISSUE TITLE
Alexa homebridge

### ISSUE THREAD

**Issue by simoneras** [mmende/homebridge-samsungtv-control2#20]
hello, alexa recognizes accessory but he doesn't answer. (google translate)

**Comment by mmende** [comment]
I don't really understand what you mean. Is this issue related to [homebridge-alexa](https://www.npmjs.com/package/homebridge-alexa)?

**Comment by simoneras** [comment]
yes, I have installed the hb-alexa plug and the plug recognizes the tv but always sees it unreachable.

**Comment by mmende** [comment]
I'll have to try the plugin myself to see what's the problem.

**Comment by simoneras** [comment]
thank you!!!👍
